concent_api/api_testing_helpers.py

```diff
@@ -80,5 +81,9 @@ def api_request(host, endpoint, private_key, public_key, data = None, headers =
     print()
 
 
+def timestamp_to_isoformat(timestamp):
+    return datetime.datetime.fromtimestamp(timestamp).isoformat()
```

Comment:
You forgot about the argument. By default it inserts a `T` between the date and time.

`isoformat(' ')`.

Please make sure the output of this function is tested in some way.

Reply:
I tested it before and it was working this way, but now I changed it to: 
`return datetime.datetime.fromtimestamp(timestamp).isoformat(' ')`